idb/postgres/postgres.go

```diff
@@ -505,10 +505,8 @@ func (db *IndexerDb) yieldTxnsThread(ctx context.Context, rows *sql.Rows, result
 			var row idb.TxnRow
 			row.Error = err
 			results <- row
-			rows.Close()
```

Comment:
I think these should have become a `defer rows.Close()`

Reply:
+1

`rows.Close()` isn't called when this function is invoked by `YieldTxns`

Reply:
done